apps/extension/src/ui/domains/Account/AccountAdd/AccountAddLedger/AddLedgerSelectNetwork.tsx

```diff
@@ -303,47 +195,61 @@ export const AddLedgerSelectNetwork = () => {
         <Spacer small />
         <AccountTypeSelector defaultType={accountType} onChange={handleTypeChange} />
         {accountType === "sr25519" && (
-          <div className="bg-black-secondary mt-12 rounded p-12">
-            <h2 className="text-body-disabled leading-paragraph text-base">
-              {t("1. Choose Ledger App Version")}
-            </h2>
-            <div className="mt-6 grid grid-cols-3 gap-8">
-              <AppVersionButton
-                title={t("Polkadot App")}
-                description={t("Supports all substrate networks")}
-                extra={
-                  <span className="bg-green/10 text-green rounded-[1.2rem] px-4 py-1">
-                    {t("Recommended")}
-                  </span>
-                }
-                selected={substrateAppType === AddSubstrateLedgerAppType.Generic}
-                onClick={handleSubstrateAppTypeClick(AddSubstrateLedgerAppType.Generic)}
-              />
-              <AppVersionButton
-                title={t("Legacy Apps")}
-                description={t("Network-specific substrate apps")}
-                selected={substrateAppType === AddSubstrateLedgerAppType.Legacy}
-                onClick={handleSubstrateAppTypeClick(AddSubstrateLedgerAppType.Legacy)}
-              />
-              <AppVersionButton
-                title={t("Migration App")}
-                description={t("Recover your assets from deprecated Ledger apps.")}
-                selected={substrateAppType === AddSubstrateLedgerAppType.Migration}
-                onClick={handleSubstrateAppTypeClick(AddSubstrateLedgerAppType.Migration)}
-                disabled={!enableMigrationApp}
-              />
+          <>
+            <div className="bg-black-secondary mt-12 rounded p-12">
+              <h2 className="text-body-secondary leading-paragraph text-base">
+                {t("1. Choose Network")}
+              </h2>
+              <div className="mt-6">
+                <SubstrateNetworkSelect chain={chain} onSelect={handleNetworkChange} />
+              </div>
             </div>
-
-            {substrateAppType === AddSubstrateLedgerAppType.Legacy && (
-              <SubstrateLegacyNetworkSelect chain={chain} onChange={handleNetworkChange} />
-            )}
-            {substrateAppType === AddSubstrateLedgerAppType.Migration && (
-              <SubstrateMigrationNetworkSelect
-                app={migrationApp}
-                onChange={handleMigrationAppChange}
-              />
+            {!!chain && (
```

Comment:
There's a slightly annoying height-shift in this component when selecting different networks because some of them have longer names, which forces the text to be 3 lines instead of 2

Reply:
ok will fix